src/cms/views/media/media_actions.py

```diff
@@ -175,8 +177,13 @@ def edit_file_ajax(request, region_slug=None):
     media_file = get_object_or_404(
         MediaFile.objects.filter(region=region), id=request.POST.get("id")
     )
-
-    media_file_form = MediaFileForm(data=request.POST, instance=media_file)
+    
+    if request.FILES:
+        # date = media_file.uploaded_date.strftime("%d/%m/%y")
+        media_file_form = MediaFileForm(data=request.POST, instance=media_file, files=request.FILES)
+        logger.warn(media_file_form.__dict__)
+    else:
+        media_file_form = MediaFileForm(data=request.POST, instance=media_file)
```

Comment:
I would suggest to create a new ajax endpoint just for handling the replacement. Also, you don't need the differentiation with `if request.FILES:`, because `Form(...)` and `Form(..., files=None)` are equivalent (if you have a look at the [source code](https://github.com/django/django/blob/93f3b7a0ba2969109940775a18515728603afac6/django/forms/forms.py#L72), you see that `files` is a keyword argument with a default value of `None`).